tests/ctst/features/iam-policies/IAMUser.feature

```diff
@@ -0,0 +1,82 @@
+Feature: IAM Policies for IAM Users
+  This feature allows you to create and attach IAM policies for IAM users.
+  IAM users should have the permissions to perform the actions that they are granted in their IAM policies.
+
+  @2.6.0
+  @PreMerge
+  @IAM-Policies-IAMUsers
+  Scenario Outline: User is not authorized to perform the actions without IAM policy
+    Given an existing bucket "<bucketName>" "without" versioning
```

Comment:
Don't we want to test both with and without versioning enabled?

```suggestion
    Given an existing bucket "<bucketName>" "<enableversioning>" versioning
```

Reply:
This might have been a typo by Xin, as indeed I think it makes sense to have this as a variable, but we'll have to see if she has a logic here.

Reply:
I am thinking that we are testing s3 iam policy auth here not functional tests of these s3 apis, versioning doesn't really have an impact on policy check, right?

Reply:
Yes, but given that we can reuse the step definitions, we can already implement a version-aware step that we could reuse in the future, even if we do not see any use case for this option today.